docs/developing-on-windows.md

```diff
@@ -0,0 +1,41 @@
+## Development on Windows 10
+
+This is a brief guide to getting a Windows environment set up for Carbon
+development. We'll be using the
+[Windows Subsystem for
+Linux](https://docs.microsoft.com/en-us/windows/wsl/about) with an Ubuntu install 👐🏽.
+
+Note: if you have native Windows versions of Yarn, NVM/Node, Python and the like
+[Windows automatically imports your paths into your WSL by default](https://github.com/Microsoft/WSL/issues/1890)
+(running `echo $PATH` can confirm this), this can be a source of weird bugs when
+WSL is attempting to run Windows versions of -- for example Python -- instead of
+your WSL version. 🤦🏽‍♂️ That interop is fine (using Windows executables via WSL),
+but can be unexpected.
+
+1. Install Ubuntu 18.04 LTS via the Microsoft Store
+2. Run the Ubuntu application -- which should look like a familiar bash terminal
+   (the remainder of the guide will happen here 👍🏽)
+3. Setup a user name and password when prompted
+4. Update your Ubuntu repositories `sudo apt-get update`
+5. Install the build-essential package `sudo apt install build-essential`
+6. Install nvm via
+   `curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.34.0/install.sh | bash`
```

Comment:
does this mean we'll need to keep this doc up to date with nvm releases?

Reply:
I debated making it more general "go install NVM" rather than "run this command: ...." but ran into lots of trouble with Yarn installed this way vs that way, NVM installed here vs there. It is technically just the `curl` command from their docs though 🤷🏽‍♂️ 

The way I see it as is if NVM is updated, they _might_ get a prompt to update NVM after its all said and done. Happy to change it though if it looks sloppy.

Reply:
not sure what issues you ran into but as you said this is a command taken from their current README so we can probably just link to nvm here since the URL updates with each release